### PR TITLE
Remove extra XXL size in  Size section and add blue in Color section

### PR DESCRIPTION
**Pull request by AntiQnis**

### CODE REVIEW

File: src/main/resources/db/MTB dml.sql [hunk 1]
```diff
@@ -10,3 +10,32 @@ values ('BIKE', 'bike'),
...
+        ('YELLOW'),
+        ('RED'),
+        ('PURPLE'),
+        ('PINK'),
+        ('GREEN'),
+        ('GRAY'),
+        ('BLUE');
+
+insert into brand (code)
+values ('NIKE'),
```
Comment (pavelvrublevskij) [inline src/main/resources/db/MTB dml.sql]: Use only Context ridewear

File: src/main/resources/db/MTB dml.sql [hunk 2]
```diff
@@ -10,3 +10,32 @@ values ('BIKE', 'bike'),
...
+        ('WHITE'),
+        ('YELLOW'),
+        ('RED'),
+        ('PURPLE'),
+        ('PINK'),
+        ('GREEN'),
+        ('GRAY'),
+        ('BLUE');
+
+insert into brand (code)
```
Comment (pavelvrublevskij) [inline src/main/resources/db/MTB dml.sql]: brand table does not have a code. Change to name here

File: src/main/resources/db/MTB dml.sql [hunk 3]
```diff
@@ -10,3 +10,32 @@ values ('BIKE', 'bike'),
...
+insert into size (code)
+values  ('XXS'),
+        ('XS'),
+        ('S'),
+        ('M'),
+        ('L'),
+        ('XL'),
+        ('XXL');
+
+insert into color (code)
```
Comment (pavelvrublevskij) [inline src/main/resources/db/MTB dml.sql]: name missed, add names for colors

File: src/main/resources/db/MTB dml.sql [hunk 4]
```diff
@@ -10,3 +10,32 @@ values ('BIKE', 'bike'),
 insert into gender (code)
 values ('MALE'),
        ('FEMALE');
+
+insert into size (code)
```
Comment (pavelvrublevskij) [inline src/main/resources/db/MTB dml.sql]: size table does not have a code. Change to value here

File: src/main/resources/templates/clothing.html [hunk 5]
```diff
@@ -101,7 +101,7 @@ <h2 class="container-selector">Size</h2>
                             <li class="cloth-size">
                                 <input class="form-check-input" type="checkbox" name="currency" id="usdCheckbox2"
                                        value="1">
-                                <label class="form-check-label" for="usdCheckbox">XS (5)</label>
+                                <label class="form-check-label" for="usdCheckbox">XS (1)</label>
```
Comment (pavelvrublevskij) [inline src/main/resources/templates/clothing.html:104]: Eliminate (1), same repeated below many times

**Review by pavelvrublevskij — CHANGES_REQUESTED** [review]
Please do changes and inform when done

File: src/main/resources/db/MTB dml.sql (hunk 4 above)
Comment (AndDauJA) [reply src/main/resources/db/MTB dml.sql]: will wait for changes

File: src/main/resources/db/MTB dml.sql (hunk 3 above)
Comment (AndDauJA) [reply src/main/resources/db/MTB dml.sql]: will wait for changes

File: src/main/resources/db/MTB dml.sql (hunk 2 above)
Comment (AndDauJA) [reply src/main/resources/db/MTB dml.sql]: will wait for changes

File: src/main/resources/db/MTB dml.sql (hunk 1 above)
Comment (AndDauJA) [reply src/main/resources/db/MTB dml.sql]: will wait for changes

File: src/main/resources/templates/clothing.html (hunk 5 above)
Comment (AndDauJA) [reply src/main/resources/templates/clothing.html:104]: will wait for changes

**Review by AndDauJA — COMMENTED** [review]
will wait for changes

File: src/main/resources/db/MTB dml.sql (hunk 1 above)
Comment (AntiQnis) [reply src/main/resources/db/MTB dml.sql]: ok ill fix it

File: src/main/resources/db/MTB dml.sql (hunk 2 above)
Comment (AntiQnis) [reply src/main/resources/db/MTB dml.sql]: ok, changed

File: src/main/resources/db/MTB dml.sql (hunk 3 above)
Comment (AntiQnis) [reply src/main/resources/db/MTB dml.sql]: fixed

File: src/main/resources/db/MTB dml.sql (hunk 1 above)
Comment (pavelvrublevskij) [reply src/main/resources/db/MTB dml.sql]: Should be -> Context ridewear

**Comment by AntiQnis** [comment]
I make all changes , Thanks for rewiew

**Review by pavelvrublevskij — CHANGES_REQUESTED** [review]
still not all comments solved

File: src/main/resources/templates/clothing.html (hunk 5 above)
Comment (AntiQnis) [reply src/main/resources/templates/clothing.html:104]: ok im done for this one also

File: src/main/resources/db/MTB dml.sql (hunk 1 above)
Comment (AntiQnis) [reply src/main/resources/db/MTB dml.sql]: ok i make changes on this on

File: src/main/resources/templates/clothing.html (hunk 5 above)
Comment (AntiQnis) [reply src/main/resources/templates/clothing.html:104]: i remove extra XS size and fix  the brand in the same clothing.html file and leave only context we have

File: src/main/resources/db/MTB dml.sql (hunk 1 above)
Comment (AntiQnis) [reply src/main/resources/db/MTB dml.sql]: ok now i made it :D finally

**Review by pavelvrublevskij — APPROVED** [review]

**Pull request merged**